scripts/change_alias.py

```diff
@@ -1,7 +1,38 @@
+from constants import INDICES, STAGING_NODE1
 from elasticsearch import Elasticsearch
-import argparse
+import click
+import logging
 
-from utils import *
+from utils import remove_underscore_from_end_prefix
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s\t%(levelname)s:\t%(name)s line %(lineno)s\t%(message)s', level=logging.INFO)
+logging.getLogger('elasticsearch').setLevel(logging.WARNING)
+
+
+@click.command()
+@click.option(
+    '--es_hosts',
+    default=STAGING_NODE1,
+    help='Specify the Elastic Search server(s) (port could be included), e.g. wp-np3-e2:9200. '
+         'If multiple servers are provided, please use ";" to separate them, e.g. "wp-np3-e2;wp-np3-e3"'
+)
+@click.option(
+    '--es_index_prefix',
+    default="",
+    help='Specify the Elastic Search index prefix, e.g. '
+         'faang_build_1 then the indices will be faang_build_1_experiment etc.'
+         'If not provided, display the current aliases'
+)
+def main(es_hosts, es_index_prefix):
```

Comment:
I don't think we should have main() method here, this code should go inside script entry point.

Reply:
could not figure out how to pass click parameter into class